components/datetime/tests/fixtures/tests/components_with_zones.json

```diff
@@ -4,16 +4,6 @@
         "input": {
             "value": "2020-01-21T08:25:07.000+05:00",
             "options": {
-                "components": {
-                    "weekday": "long",
-                    "month": "long",
-                    "day": "numeric-day-of-month",
-                    "year": "numeric",
-                    "hour": "two-digit",
-                    "minute": "two-digit",
-                    "second": "two-digit",
-                    "time_zone_name": "long-generic"
-                },
```

Comment:
So, I had mixed feelings about deleting these, which is why I hadn't done it yet. I do think it is potentially useful, if we ever add classical skeletons back again, to have these fixtures, even though they are dead right now.